packages/frontend-2/lib/common/helpers/route.ts

```diff
@@ -21,6 +21,71 @@ export const defaultZapierWebhookUrl =
   'https://hooks.zapier.com/hooks/catch/12120532/2m4okri/'
 export const guideBillingUrl = 'https://speckle.guide/workspaces/billing.html'
 
+export const settingsRoutes = {
+  user: {
+    profile: {
```

Comment:
Both route and name are needed, because they are used in different scenarios (for example checking if a route by name needs SSO access)